FFMQRLib/Enemies.cs

```diff
@@ -163,11 +191,64 @@ public void Write(FFMQRom rom)
             rom.PutInBank(RomOffsets.EnemiesStatsBank, RomOffsets.EnemiesStatsAddress + (_Id * RomOffsets.EnemiesStatsLength), _rawBytes);
         }
     }
-    public class EnemiesAttacks
+    public class EnemyAttack
```

Comment:
EnemyScript might be more appropriate a name and would avoid confusion with the actual attacks

Reply:
Renamed it to EnemyAttackLink(s), from the SQL world, describing a many-to-many relationship. Let me know what you think.